.gitmodules

```diff
@@ -36,7 +36,7 @@ fxDONOTUSEurl = https://github.com/NGEET/fates
 [submodule "cism"]
 path = components/cism
 url = https://github.com/ESCOMP/CISM-wrapper
-fxtag = cismwrap_2_2_010
+fxtag = cismwrap_2_2_011
```

Comment:
Wait this is a change on top of the update to alpha-ctsm5.4.CMIP7.21.ctsm5.3.085. This should be done as a separate step. Or maybe this shows up because this is based to master and not post-5.4?

Reply:
Yep, that's why it showed up. Now it doesn't.